### DISCUSSION TITLE
inbucket service is active but mail not receiving showing bellow error

### DISCUSSION THREAD

**Discussion by techsachin1** [inbucket/inbucket#368] [Q&A]
hi,
my inbucket service is running & Active. but showing this error bellow,
does it matter for mail receiving & how to resolve it if its matter.

http: superfluous response.WriteHeader call from github.com/inbucket/inbucket/pkg/server/web.Handler.ServeHTTP s response.WriteHeader call from github.com/inbucket/inbucket/pkg/server/web.Handler.ServeHTTP

Error handling request error="websocket: the client is not using the websocket protocol: 'upgrade' token not found in error="websocket: the client is not using the websocket protocol: 'upgrade' token not found in 'Connection' header" module=web path=/api/v1/monitor/messages

**Comment by jhillyerd** [comment]
This won't prevent Inbucket from receiving mail, but it may prevent the `Monitor` tab from displaying the list of recently received messages.

#103 and #96 may provide some hints for websockets on nginx
